SwrveConversationSDK/Conversation/SwrveConversationItemViewController.m

```diff
@@ -63,6 +64,86 @@ - (void)viewDidDisappear:(BOOL)animated {
     }
 }
 
+- (void)dealloc {
+    
+    [[NSNotificationCenter defaultCenter] removeObserver:self
+                                                    name:kSwrveNotificationViewReady
+                                                  object:nil];
+    [[NSNotificationCenter defaultCenter] removeObserver:self
+                                                    name:UIDeviceOrientationDidChangeNotification
+                                                  object:nil];
+    [[UIDevice currentDevice] endGeneratingDeviceOrientationNotifications];
```

Comment:
that's only if viewDidLoad is guaranteed to be called again every time a conversation is opened and it doesn't appear to be. Dealloc is when the view is truly done with it.